apps/evm/lib/evm/message_call.ex

```diff
@@ -105,10 +106,19 @@ defmodule EVM.MessageCall do
         message_call.current_exec_env
         |> Map.put(:account_interface, n_exec_env.account_interface)
 
+      refund = n_sub_state.refund + message_call.current_sub_state.refund
+
+      sub_state = %SubState{
+        refund: n_sub_state.refund + message_call.current_sub_state.refund,
```

Comment:
Could we use the `refund` that is calculated in line `109` here? I think it's the same calculation, right?